akka-distributed-data/src/test/resources/reference.conf

```diff
@@ -1,2 +1,5 @@
-akka.actor.serialize-messages = on
-akka.actor.serialize-creators = on
+akka.actor {
+  serialize-messages = on
+  serialize-creators = on
+  warn-about-java-serializer-usage = off
+}
```

Comment:
missing newline here can break multi-node test, I'm not sure if this is included in those